katello-attach-subscription

```diff
@@ -967,9 +982,80 @@ def vdcupdate()
         end
 
         if @options[:emptyhypervisor]
-          skip_hypervisor ||= !hypervisor_has_guests(sys)
+          skip_host ||= !hypervisor_has_guests(sys)
+        end
+      end
+
+      derived_sub = {}
+      attachable_subs = []
+      # if system is a guest and density option is on, check if use_derived is true
+      if system_type == KatelloAttachSubscription::FactAnalyzer::GUEST and @options[:density] and use_derived
+        # if so, fetch the stack-derived sub available for that host and replace the one in sub_parsing
+        if @options[:verbose]
+          puts "   VERBOSE: System #{system['name']} is a Guest and desire stack-derived sub, search for it"
+        end
+        # subscriptions?available_for=host&host_id=HOST_ID
```

Comment:
this comment is not telling much, and if, should be at line 1009 with `attachable_subs`? but I'd just drop it :)